.github/pull_request_template.md

```diff
@@ -1,9 +1,12 @@
-**Description:** <Describe what has changed.>
 <!--Ex. Fixing a bug - Describe the bug and how this fixes the issue.
 Ex. Adding a feature - Explain what this achieves.-->
+Description:
 
-**Link to tracking Issue:** <Issue number if applicable>
```

Comment:
I'd still like to have the section header here, but it would be good to move the comment to be hidden from the displayed md contents 👍

Reply:
I personally could go any way here. Having it as part of the description makes sense, but giving it its own section works too. I'd also be happy with putting `Fixes #` outside of the sections and having it at the top or bottom of the template.

Reply:
I realize I'm being pretty nitpicky here, I'll defer to others if there's a general preference or strong opinion.